tests/library/selector-generator.spec.ts

```diff
@@ -133,24 +133,26 @@ it.describe('selector generator', () => {
     await page.setContent(`
       <div>Hello world</div>
       <a>Hello <span>world</span></a>
+      <a>Goodbye <span>world</span></a>
     `);
-    expect(await generate(page, 'a')).toBe(`a:has-text("Hello world")`);
+    expect(await generate(page, 'a:has-text("Hello")')).toBe(`a:has-text("Hello world")`);
   });
 
   it('should chain text after parent', async ({ page }) => {
     await page.setContent(`
       <div>Hello <span>world</span></div>
       <a>Hello <span mark=1>world</span></a>
+      <a>Another A</a>
     `);
-    expect(await generate(page, '[mark="1"]')).toBe(`a >> text=world`);
+    expect(await generate(page, '[mark="1"]')).toBe(`a:has-text(\"Hello world\")`);
```

Comment:
I think you should change `a`s in this test to `div`s to preserve the intent of the test.